src/libsnore/snore.cpp

```diff
@@ -102,7 +102,7 @@ void SnoreCore::loadPlugins(SnorePlugin::PluginTypes types)
             }
         }
     }
-    QMetaObject::invokeMethod(d, "slotInitPrimaryNotificationBackend", Qt::QueuedConnection);
+    QMetaObject::invokeMethod(d, "slotInitPrimaryNotificationBackend", Qt::DirectConnection); // Direct connection is used because the notification backend should be available after plugins are loaded 
```

Comment:
Yes, you're right. I changed it because when I tried to run simple app that only displays notification and discovered that notification is not displayed because notification backend was not initialized yet (even if I executed loadPlugins).